cfme/common/__init__.py

```diff
@@ -338,15 +335,19 @@ def remove_tags(self, tags):
         """Remove multiple of tags
 
         Args:
-            tags: pass dict with category name as key, and tag as value,
-                 or pass list with tag objects
+            tags: list of tag objects
         """
-        if isinstance(tags, dict):
-            for category, tag in tags.items():
-                self.remove_tag(category=category, tag=tag)
-        elif isinstance(tags, (list, tuple)):
-            for tag in tags:
-                self.remove_tag(tag=tag)
+        for tag in tags:
+            self.remove_tag(tag=tag)
+
+    def _set_random_tag(self, view):
+        random_cat = random.choice(view.form.tag_category.all_options).text
+        random_cat_cut = random_cat[:-1].strip() if random_cat[-1] == '*' else random_cat
```

Comment:
**SUGGESTION:** Add comment to explain why * would be at the end of category name.